datadog_checks_dev/tests/tooling/commands/validate/test_jmx.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+from datadog_checks.dev.tooling.commands.validate.jmx_metrics import duplicate_bean_check, jmx_metrics
+
+regular_beans = {
+  "jmx_metrics": [
+    {
+      "include": {
+        "attribute": {
+          "Count": {
+            "metric_type": "gauge", 
+            "alias": "foo.bar"
+          }
+        }, 
+        "bean": "my.bean:type=foo,name=baz"
+      }
+    }, 
+    {
+      "include": {
+        "attribute": {
+          "Rate": {
+            "metric_type": "gauge", 
+            "alias": "foo.bar"
+          }
+        }, 
+        "bean": "my.bean:type=foo,name=baz"
+      }
+    }
+  ]
+}
+duplicate_beans = {
+  "jmx_metrics": [
+    {
+      "include": {
+        "attribute": {
+          "Count": {
+            "metric_type": "gauge", 
+            "alias": "foo.bar"
+          }
+        }, 
+        "bean": "my.bean:type=foo,name=baz"
+      }
+    }, 
+    {
+      "include": {
+        "attribute": {
+          "Count": {
+            "metric_type": "gauge", 
+            "alias": "foo.bar"
+          }
+        }, 
+        "bean": "my.bean:type=foo,name=baz"
+      }
+    }
+  ]
+}
+
+@pytest.mark.parametrize(
+    'beans,errors',
+    [
+        (regular_beans, 0),
+        (duplicate_beans, 1),
+    ],
+)
+def test_duplicate_beans(beans, errors):
+    result = duplicate_bean_check(beans.get("jmx_metrics"))
+    assert len(result) >= errors
```

Comment:
nit- I think it would be more helpful to test for the actual error messages. Is there a reason this doesn't?